docs/index.md

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_label: 'Quick start'
+---
+
+# Apify API client for JavaScript
+`apify-client` is the official library to access [Apify API](https://docs.apify.com/api/v2) from your
+JavaScript applications. It runs both in Node.js and browser and provides useful features like
+automatic retries and convenience functions that improve the experience of using the Apify API.
+
+## Quick Start
+```js
+const { ApifyClient } = require('apify-client');
```

Comment:
```suggestion
import { ApifyClient } from 'apify-client';
```

i guess we could start using the import statements to be consistent with the rest, e.g. sdk does it already